Update readme files (#236)
membraneframework/membrane_demo: webrtc_authentication/README.md

# Membrane Demo - WebRTC Singaling Server

An example of signaling server with authentication based on `Membrane.WebRTC.Server`.

## Prerequisites

Make sure you have `postgresql` installed on your computer.

### MacOS

```console
brew install postgresql
```

Then run the database server:

```console
brew services start postgresql
```

### Ubuntu

```console
sudo apt-get install postgresql
```

Then run the database server:

```console
sudo service postgresql start
```

Furthermore, make sure you have Elixir installed on your machine. For installation details, see: https://elixir-lang.org/install.html

On Ubuntu, we recommend installation through asdf, see: https://asdf-vm.com/guide/getting-started.html

## Running the demo

To run the demo, clone the `membrane_demo` repository and checkout to the demo directory:

```console
git clone https://github.com/membraneframework/membrane_demo
cd membrane_demo/webrtc_authentication
```

> Custom ip, port or other Plug options can be set up in `config/config.exs`.

Reviewer: ```suggestion
Custom database's IP, port, name and other `Plug` options can be set up in `config/config.exs`.
```

Then you need to download the dependencies of the mix project:

```
mix deps.get
```

Then you may be asked to install `Hex` and then `rebar3`.

In case of installation issues with Hex on Ubuntu, try updating the system packages first by entering the command:

```shell
sudo apt-get update
```

### Guardian

This application uses [Guardian](https://github.com/ueberauth/guardian) to authenticate
the users. Generate your secret key with:

```
mix guardian.gen.secret
```

and add it to the config file (`config/config.exs`).

### Database

Configure a database for the application by running:

```
psql -d postgres
```

and then in the psql console:

```
\du
```

> Change `username` in `config/config.exs` to the name of the user you have in table.

Reviewer: I would say that describing how to check the username in postgres shouldn't be in the scope of that readme. Perhaps it would be better just to once again emphasize, that the user should set their databases configuration in `config/config.exs`, WDYT?

Author: Your suggestion is ok, i will change it as you suggest

### Ubuntu

```shell
apt install ca-certificates
cp priv/certs/certificate.pem /usr/local/share/ca-certificates/
update-ca-certificates
```

### Arch

```shell
trust anchor --store priv/certs/certificate.pem
```

### MacOS

shell

> 

Reviewer: ```suggestion
```
I think there shouldn't be that empty line
